src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.ts

```diff
@@ -1,57 +1,66 @@
+import { DotActionMenuItem } from '@models/dot-action-menu/dot-action-menu-item.model';
+import { DotTemplatesService } from '@services/dot-templates/dot-templates.service';
+import { DotMessageDisplayService } from '@components/dot-message-display/services';
+import { DotMessageSeverity, DotMessageType } from '@components/dot-message-display/model';
+import { DotPushPublishDialogService } from 'dotcms-js';
 import { DotRouterService } from '@services/dot-router/dot-router.service';
-import { ActionHeaderOptions } from '@shared/models/action-header';
-import { DotTemplate } from '@shared/models/dot-edit-layout-designer';
+import { DialogService } from 'primeng/dynamicdialog';
+import { DotBulkInformationComponent } from '@components/_common/dot-bulk-information/dot-bulk-information.component';
+import {
+    DotActionBulkResult,
+    DotBulkFailItem
+} from '@models/dot-action-bulk-result/dot-action-bulk-result.model';
 
 @Component({
     selector: 'dot-template-list',
     templateUrl: './dot-template-list.component.html',
     styleUrls: ['./dot-template-list.component.scss']
 })
 export class DotTemplateListComponent implements OnInit, OnDestroy {
-    actions: ActionHeaderOptions;
-    firstPage: DotTemplate[];
+    @ViewChild('listing', { static: false })
+    listing: DotListingDataTableComponent;
     tableColumns: DataTableColumn[];
+    firstPage: DotTemplate[];
+    templateBulkActions: MenuItem[];
+    actionHeaderOptions: ActionHeaderOptions;
+    addToBundleIdentifier: string;
+    selectedTemplates: DotTemplate[] = [];
 
+    private isEnterPrise: boolean;
+    private hasEnvironments: boolean;
     private destroy$: Subject<boolean> = new Subject<boolean>();
 
     constructor(
         private route: ActivatedRoute,
         private dotMessageService: DotMessageService,
-        private dotRouterService: DotRouterService
+        private dotTemplatesService: DotTemplatesService,
+        private dotMessageDisplayService: DotMessageDisplayService,
+        private dotPushPublishDialogService: DotPushPublishDialogService,
+        private dotRouterService: DotRouterService,
+        public dialogService: DialogService
     ) {}
 
     ngOnInit(): void {
-        this.actions = {
-            primary: {
-                model: [
-                    {
-                        command: () => {
-                            this.dotRouterService.gotoPortlet('/templates/new/designer');
-                        },
-                        label: 'Designer'
-                    },
-                    {
-                        command: () => {
-                            this.dotRouterService.gotoPortlet('/templates/new/advanced');
-                        },
-                        label: 'Advanced'
-                    }
-                ]
-            }
-        };
-
         this.route.data
             .pipe(pluck('dotTemplateListResolverData'), takeUntil(this.destroy$))
```

Comment:
```suggestion
            .pipe(pluck('dotTemplateListResolverData'), take(1))
```

This will happen only once anyways.